src/components/Button/PlayButton.js

```diff
@@ -30,7 +30,7 @@ const StyledPlayButton = styled.div`
   ${props => props.hover ? 'opacity: 0;' : ''}
   ${props => 
     props.type === 'lesson' ||
-    (props.type === 'course' && (props.expanded === false || props.expanded === 'horizontal'))
+    (props.type === 'course' && (!props.expanded || props.expanded === 'horizontal'))
```

Comment:
When I refactored the `CourseCard` expansions to remove `false` in favor of only being `['horizontal', 'vertical']`, I missed updating this line away from `=== false` to a falsey `undefined` check.